README.md

```diff
@@ -59,11 +65,11 @@ For example `ca mycafile.pem` becomes
 ```
 Footnotes
 -----------
-Please note that OpenVPN used by this project is under GPLv2. 
+Please note that OpenVPN used by this project is under GPLv2+.
 
-If you cannot or do not want to use the Play Store you can [download the apk files directly](http://plai.de/android/).
+If you cannot or do not want to use the Play Store you can [download the APK files directly](http://plai.de/android/).
```

Comment:
apk is written in lowercase, https://en.wikipedia.org/wiki/Apk_(file_format)

Reply:
That is a mistake on Wikipedia. The rest of the article has it right.
Possibly it was meant to be ".apk" in the heading. (Which is a possible choice here too.)
Also, https://developer.android.com/guide/components/fundamentals